app/Exceptions/ZipException.php

```diff
@@ -3,7 +3,7 @@
 class FreshRSS_Zip_Exception extends Exception {
 	private $zipErrorCode = 0;
 
-	public function __construct($zipErrorCode) {
+	public function __construct($zipErrorCode, $url = null) {
 		parent::__construct('ZIP error! ' . $url, 2141);
```

Comment:
```suggestion
	public function __construct($zipErrorCode) {
		parent::__construct('ZIP error!', 2141);
```

I believe this is what @aledeg suggests. You can just click on accept here @ColonelMoutarde